lib/plugins/aws/custom-resources/index.js

```diff
@@ -179,6 +179,19 @@ async function addCustomResourceToService(awsProvider, resourceName, iamRoleStat
   };
   Resources[customResourceFunctionLogicalId] = customResourceFunction;
 
+  // configure vpc for the custom resource function
+  providerConfig.vpc = providerConfig.vpc || { securityGroupIds: null, subnetIds: null };
```

Comment:
Sorry just spotted it now, let's not modify `providerConfig.vpc`, downstream it may create some unwanted behavior (e.g. with plugins).

Let's simply extend `customResourceFunction.Properties` only if there's a `vpc` configuration on `provider`